deployability/modules/testing/models.py

```diff
@@ -2,22 +2,20 @@
 from pydantic import BaseModel, field_validator, model_validator
 from typing import Literal
 
-
 class ExtraVars(BaseModel):
     """Extra vars for testing module."""
     component: Literal['manager', 'agent']
-    dependency: str | None = None
+    dependencies: list[str]
     wazuh_version: str
     wazuh_revision: str
     wazuh_branch: str | None = None
     working_dir: str = '/tmp/tests'
 
-
 class InputPayload(ExtraVars):
     """Input payload for testing module."""
     tests: list[str]
     inventory: Path
-    dependency: Path = None
+    dependencies: list[str]
```

Comment:
This must be nullable or have a default value, because some test could not require a dependency, so this list would be empty

```suggestion
    dependencies: list[str] = []
```
This way, when the `dependencies` attribute is not received, its defaults on []

Reply:
Done https://github.com/wazuh/wazuh-qa/pull/4884/commits/507e376cb389ea15245b6538a710341f4cdf0ec1